src/ICeWall/doc/set_bump.md

```diff
@@ -0,0 +1,38 @@
+## set_bump_options
+
+### Synopsis
```

Comment:
can't believe that didn't trigger the security scan!

Reply:
The company is Synops**y**s.  The script doesn't check for EDA vendor names today (though maybe it should), just PDK providers